src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.Reflection.cs

```diff
@@ -298,13 +298,18 @@ private static bool IsRelevantNamespace(string? xamlNamespace)
 			return true;
 		}
 
-		private static bool IsRelevantProperty(XamlMember? member)
+		private static bool IsRelevantProperty(XamlMember? member, XamlObjectDefinition objectDefinition)
 		{
 			if (member?.Name == "Phase") // Phase is not relevant as it's not an actual property
 			{
 				return false;
 			}
 
+			if (objectDefinition.Type.Name == "Style" && member?.Name == "IsNativeStyle")
```

Comment:
Would switching this validation bring any performance gain? Having into consideration that `member` could be null
```suggestion
			if (member?.Name == "IsNativeStyle" && objectDefinition.Type.Name == "Style")
```

Reply:
Theoretically speaking, yes, considering that `objectDefinition.Type.Name` being "Style" is more common than `member?.Name` being "IsNativeStyle". However, it will be hard to notice the difference in practice. I'm going to take the suggestion.

Reply:
Was just a simple comment, but thanks for validating it.